src/components/Blocking/ItemList.js

```diff
@@ -40,7 +41,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: 14,
     paddingHorizontal: 20,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous one, replacing a hard-coded color value with a dynamic one from the theme object. This is a good practice for the same reasons mentioned above.
```diff
- color: '#000',
+ color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->